core/serv-api/src/lib.rs

```diff
@@ -55,14 +56,49 @@ impl CommandOutput {
                 header,
             } => {
                 if json_output {
-                    println!(
-                        "{}",
-                        serde_json::to_string_pretty(&serde_json::json!({
-                            "headers": columns,
-                            "values": values
-                        }))
-                        .unwrap()
-                    )
+                    let columns_size_eq_values_size = values
+                        .iter()
+                        .map(|high_value| {
+                            if let serde_json::Value::Array(values) = high_value {
+                                values.len()
+                            } else {
+                                0
+                            }
+                        })
+                        .filter(|size| *size != columns.len())
+                        .collect::<Vec<usize>>()
+                        .len()
+                        == 0;
```

Comment:
shorter version without `collect`:

```rust
let columns_size_eq_values_size = !values.iter().any(|row| match row {
    serde_json::Value::Array(values) => values.len() != columns.len(),
     _ => true,
});
```

Reply:
Or maybe this to avoid negating twice?
```rust
let columns_size_eq_values_size = values.iter().all(|row| match row {
    serde_json::Value::Array(values) => values.len() == columns.len(),
     _ => false,
});

```